src/components/Community/CommunitySidebar.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState } from "react";
+
+import { styled } from "styled-components";
+
+import {
+  ImgHouse,
+  ImgWorkshop,
+  ImgProject,
+  ImgCommuity,
+  ImgContents,
+  ImgDesign,
+  ImgOpen,
+  IcArrowdownSm,
+} from "../../assets";
+
+import Dropdown from "./Dropdown";
+
+function CommunitySidebar() {
+  const navCategory = [
+    { content: "홈", element: <ImgHouse />, dropdown: false },
+    { content: "워크숍&세미나", element: <ImgWorkshop />, dropdown: false },
+    { content: "프로젝트&챌린지", element: <ImgProject />, dropdown: false },
+    { content: "커뮤니티", element: <ImgCommuity />, dropdown: true },
+    { content: "콘텐츠", element: <ImgContents />, dropdown: false },
+  ];
+
+  const outLinkCategory = [
+    { content: "디자이너 생존 테스트", element: <ImgDesign /> },
+    { content: "디자이너 오픈 채팅방", element: <ImgOpen /> },
+  ];
+
+  const commuityCategory = [
+    "전체",
+    "자유주제",
+    "포트폴리오",
+    "취업/이직/커리어",
+    "사이드 프로젝트/스터디",
+    "내 생존 유형은?",
+  ];
+
+  const [btnActive, setBtnActive] = useState(0);
+  const [toggleOpen, setToggleOpen] = useState(false);
+
+  const navBtnActive = (e: number) => {
+    setBtnActive(e);
+  };
+
+  const toggleBtnActive = () => {
+    setToggleOpen(!toggleOpen);
+  };
+
+  return (
+    <CommunutySidebarWrapper>
+      <SidebarContainer>
+        {navCategory.map((item, idx) => (
+          <SidebarBtnContainer key={item.content}>
+            <SidebarBtn
+              type="button"
+              key={item.content}
+              value={idx}
+              className={"btn" + (idx == btnActive ? " active" : "")}
+              // 현재는 홈에 바탕색 고정. 클릭마다 바탕색 이벤트를 주고 싶을 경우 0 >> idx로 변경
+              onClick={() => {
+                navBtnActive(0);
+                {
+                  item.dropdown === true ? toggleBtnActive() : null;
+                }
+              }}>
+              <NavImg>{item.element}</NavImg>
+              {item.content}
+              {item.dropdown === true ? (
+                <ToggleBtn type="button" className={"btn" + (toggleOpen == true ? " active" : "")} />
+              ) : null}
+            </SidebarBtn>
+            {item.dropdown === true ? (
+              <Dropdown isOpen={toggleOpen}>
+                {commuityCategory.map((item) => {
+                  return (
+                    <DropdownBtn type="button" key={item}>
+                      {item}
+                    </DropdownBtn>
+                  );
+                })}
+              </Dropdown>
+            ) : null}
+          </SidebarBtnContainer>
+        ))}
+      </SidebarContainer>
+      <OutLinkBtnContainer>
+        {outLinkCategory.map((item) => {
+          return (
+            <OutLinkBtn type="button" key={item.content}>
+              <OutLinkImg>{item.element}</OutLinkImg>
+              {item.content}
+            </OutLinkBtn>
+          );
+        })}
```

Comment:
이 부분도
```suggestion
        {outLinkCategory.map((item) => 
          (
            <OutLinkBtn type="button" key={item.content}>
              <OutLinkImg>{item.element}</OutLinkImg>
              {item.content}
            </OutLinkBtn>
          )
        )}
```

이렇게 해줄 수 있습니다!!!